src/proxy.rs

```diff
@@ -9,6 +9,8 @@ use std::{future::Future, pin::Pin};
 use tower::{Layer, Service};
 use tracing::debug;
 
+const PROXY_METHODS: [&str; 2] = ["engine_", "eth_sendRawTransaction"];
```

Comment:
```suggestion
const MULTIPLEX_METHODS: [&str; 2] = ["engine_", "eth_sendRawTransaction"];
```
I would prefer to call it multiplex since it is the name we have been using so far for these methods.